Documentation/Settings.cfg

```diff
@@ -23,6 +23,19 @@ project_discussions  =
 
 use_opensearch       =
 
+[sphinx_object_types_to_add]
+
+data-processor-csv =  data-processor-csv // data-processor-csv // Data processor CommaSeparatedValueProcessor
```

Comment:
Can you add a comment what this syntax means?

Reply:
As this is a general new feature provided by the new rendering container it should be documented in How to document, not in each and every place we use it?

Reply:
I am fine, as above is the comment where to find that syntax (https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/GeneralConventions/FileStructure.html#settings-cfg). Can be add this information there first? Want to understand before approving the PRs.